bundle/manifests/oran-o2ims.clusterserviceversion.yaml

```
@@ -135,8 +135,27 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    containerImage: quay.io/openshift-kni/oran-o2ims
```

With https://github.com/openshift/release/pull/54191 and https://github.com/openshift-kni/oran-o2ims/pull/187, the remaining image is oran-o2ims-operator

```suggestion
    containerImage: quay.io/openshift-kni/oran-o2ims-operator
```